developer-workflow/communication-channels.rst

```diff
@@ -27,45 +27,26 @@ in return.
-developers plus many people simply interested in following Python's
-development. Discussion is focused on issues related to Python's development,
-such as how to handle a specific issue, a PEP, etc.
-
-- Ideas about new functionality should **not** start here and instead
-  should be sent to python-ideas_.
-- Technical support questions should also not be asked here and instead
-  should go to python-list_ or python-help_.
-
-Python-ideas_ is a mailing list open to the public to discuss ideas on changing
-Python. If a new idea does not start here (or python-list_, discussed below),
-it will get redirected here.
-
-Sometimes people post new ideas to python-list_ to gather community opinion
-before heading to python-ideas_. The list is also sometimes known as
-comp.lang.python, the name of the newsgroup it mirrors (it is also known by
-the abbreviation c.l.py).
-
-The python-committers_ mailing list is a private mailing list for core
-developers (the archives are publicly available).
-If something only affects core developers (e.g., the
-tree is frozen for commits, etc.), it is discussed here instead of python-dev
-to keep traffic down on the latter.
-
-python-dev_, python-committers_, and python-ideas_ all use Mailman 3, and
-are hence accessible via the Mailman 3 `web gateway`_.
-
-Python-checkins_ sends out an email for every commit to Python's various
-repositories from https://github.com/python/cpython. All core developers
-subscribe to this list and are known to reply to these emails to make comments
-about various issues they catch in the commit. Replies get redirected to
-python-dev.
-
-There are two mailing lists related to issues on the `issue tracker`_. If you
-only want an email for when a new issue is open, subscribe to
-new-bugs-announce_. If you would rather receive an email for all changes made
-to any issue, subscribe to python-bugs-list_.
+.. note:: Some mailing lists have been supplanted by categories in the
+  Python `Discourse`_. Specifically,
+
+   * The python-dev list is superseded by the `Core Development`_
+     and `PEPs`_ categories on Discourse.
+
+   * The python-ideas list is superseded by posts in the `Ideas`_
+     category on Discourse.
+
+   Discussion in :guilabel:`Core Development` is focused on issues related to Python's
+   own development, such as how to handle a specific issue, a PEP, etc.
+
+   - Ideas about new functionality should **not** start here, and instead
+     should be discussed in `Ideas`_.
+   - Technical support questions should also not be asked here, and instead
+     should go to the python-list_ or python-help_ mailing lists, or the
+     `Python Help`_ category on Discourse.
+
+Existing threads on the python-dev_, python-committers_, and python-ideas_ mailing lists
+can be accessed through the `online archive <web gateway_>`__.
 
 General Python questions should go to `python-list`_ or `tutor`_
 or similar resources, such as StackOverflow_ or the ``#python`` IRC channel
```

Comment:
I can't reach it, but it might be worth updating the links in the "complete list of Python mailing lists" since it still points to the Mailman 2 instance, when all but a few active lists have been migrated to Mailman 3 by now.

Reply:
I'll leave the Mailman2 link there, but add the Mailman3 link.